formatTest/typeCheckedTests/expected_output/comments.re

```diff
@@ -21,43 +34,44 @@
 /* comment ***/
 /* comment ****/
 /* comment *****/
+
 /**
  * Multiline
  */
+
 /** Multiline
  *
  */
+
 /**
  **
  */
-
 let testingNotQuiteEndOfLineComments = [
-  "Item 1", /* Comment For First Item */
-  "Item 2", /* Comment For Second Item */
-  "Item 3", /* Comment For Third Item */
+  "Item 1" /* Comment For First Item */,
+  "Item 2" /* Comment For Second Item */,
+  "Item 3" /* Comment For Third Item */,
   "Item 4" /* Comment For Fourth Item - but no semi */
   /* Comment after last item in list. */
-];
+] /* Comment after list bracket */;
 
-/* Comment after list bracket */
 let testingEndOfLineComments = [
   "Item 1", /* Comment For First Item */
   "Item 2", /* Comment For Second Item */
   "Item 3", /* Comment For Third Item */
   "Item 4" /* Comment For Fourth Item - but before semi */
   /* Comment after last item in list. */
-];
+] /* Comment after list bracket */;
 
-/* Comment after list bracket */
 /* This time no space between bracket and comment */
-let testingEndOfLineComments = [];
+let testingEndOfLineComments =
+  [] /* Comment after list bracket */;
 
-/* Comment after list bracket */
-type t = (int, int); /* End of line on t */
+type t = (int, int /* End of line on t */);
```

Comment:
This is a regression which I can't get right. The input is 
![image](https://cloud.githubusercontent.com/assets/1017689/17004838/d2570f6e-4e8b-11e6-85d7-bea86a4af10d.png)
and the algorithm is confused about where it should attach the comment to.